README.md

```diff
@@ -17,16 +11,31 @@ More details can be found in our preprint describing the Viv library and related
 
 > Trevor Manz, Ilan Gold, Nathan Heath Patterson, Chuck McCallum, Mark S Keller, Bruce W Herr II, Katy Börner, Jeffrey M Spraggins, Nils Gehlenborg, "[Viv: Multiscale Visualization of High-resolution Multiplexed Bioimaging Data on the Web](https://osf.io/wd2gu/)." **OSF Preprints** (2020), [doi:10.31219/osf.io/wd2gu](https://doi.org/10.31219/osf.io/wd2gu)
 
-## Avivator
+### Related Software 
+
+  - **Avivator** Included in this repository is [`Avivator`](http://avivator.gehlenborglab.org), a lightweight viewer for remote imaging data. Avivator is a purely client-side program that only requires access to [Bio-Formats-compatiable Zarr](./tutorial#option-1-create-a-bio-formats-raw-zarr) or OME-TIFF data over HTTP or on local disk.
+  - **Vizarr** [Vizarr](https://github.com/hms-dbmi/vizarr) is a minimal, purely client-side program for viewing Zarr-based images built with Viv. It exposes a Python API using the [imjoy-rpc](https://github.com/imjoy-team/imjoy-rpc) and can be directly embedded in Jupyter Notebooks or Google Colab Notebooks.
+  - **Viv benchmark** A set of scripts to benchmark Viv's retrieval of image tiles from pyramidal OME-TIFF files and Zarr stores via HTTP1 and HTTP2.
+
+## Supported Data Formats
+
 
-Also included in this repository is [`Avivator`](http://avivator.gehlenborglab.org), a lightweight "batteries-included" 
-WebGL viewer for remote imaging data. Avivator is a purely client-side program that only requires access to
-Bio-Formats "raw" Zarr or OME-TIFF data over HTTP. To use Avivator with your own data, please see the data preparation 
-[tutorial](tutorial/README.md). Initial load time for OME-TIFFs can be optimized by generating a special `offsets.json`
-file containing byte offsets for the associated binary data. For more information, see the 
-[documentation](http://viv.gehlenborglab.org/#data-preparation).
+Viv supports a subset of formats that can be generated with the [`bioformats2raw` + `raw2ometiff` pipeline]((https://www.glencoesoftware.com/blog/2019/12/09/converting-whole-slide-images-to-OME-TIFF.html)):
 
+- OME-TIFF files (pyramidal)
+- Bioformats-compatible Zarr stores (pyramidal)
 
+Please see the [tutorial](./tutorial/README.md) for more information on these formats.
```

Comment:
We also support normal OME-TIFF files (i.e non-pyramidal).  I will make a separate PR and request your review @ngehlenborg @manzt  but it was brought to my attention recently that `raw2ometiff` is actually just an implementation of a larger spec introduced [here](https://docs.openmicroscopy.org/ome-model/6.2.0/ome-tiff/specification.html#sub-resolutions) and implemented in other command line tools as well, such as the more ubiquitous `bfconvert`.

Reply:
Thanks!